astropy/io/fits/tests/test_table.py

```diff
@@ -976,7 +975,7 @@ def test_add_data_by_rows(self):
         assert tbhdu.columns.columns[2].array[0] == ''
         assert (tbhdu.columns.columns[3].array[0] ==
                 np.array([0., 0., 0., 0., 0.], dtype=np.float32)).all()
-        assert tbhdu.columns.columns[4].array[0] == True
+        assert tbhdu.columns.columns[4].array[0] == True  # nopep8
```

Comment:
I meant to just remove `== True` so that it just asserts truthiness, i.e.,
```
assert tbhdu.columns.columns[4].array[0]
```
you do that in all the lines below too, so I think is consistent.